ui/src/helpers/get-status-class.ts

```diff
@@ -23,12 +23,15 @@ import { JobStatus, OrtRunStatus, Severity } from '@/api/requests';
 export type Status = JobStatus | OrtRunStatus | undefined;
 
 // There is no vulnerability rating in the OpenApi spec, so define it here.
-export type VulnerabilityRating =
-  | 'CRITICAL'
-  | 'HIGH'
-  | 'MEDIUM'
-  | 'LOW'
-  | 'NONE';
+export const VulnerabilityRatings = {
+  CRITICAL: { text: 'CRITICAL', value: 4 },
+  HIGH: { text: 'HIGH', value: 3 },
+  MEDIUM: { text: 'MEDIUM', value: 2 },
+  LOW: { text: 'LOW', value: 1 },
+  NONE: { text: 'NONE', value: 0 },
```

Comment:
Is there a way in TypeScript to avoid the repetition of names and explicit definition of ordinal values? Like, in Kotlin we'd just use an `enum class` and use the `name` and `ordinal` properties for each entry.

Reply:
Yes I believe there is, let me fix.

Reply:
I didn't check if this is used in other ways, but maybe you could make it a simple const object:

```typescript
export const VulnerabilityRatings = {
  CRITICAL: 4,
  HIGH: 3,
  MEDIUM: 2,
  LOW: 1,
  NONE: 2,
} as const
```

Reply:
@Etsija, [enums are generally recommended to be avoided](https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh), so I would use the above-mentioned object instead.

Reply:
Done. Also decided to fix one DOM error while I'm at it.